packages/web3-eth-accounts/types/index.d.ts

```diff
@@ -83,20 +83,7 @@ export interface EncryptedKeystoreV3Json {
     }
 }
 
-/**            MOVE ALL BELOW TO WEB3-CORE ONCE TYPES COMPLETE FOR CLEAR UP     !!! */
-export interface Transaction {
-    from?: string | number;
-    to?: string;
-    gasPrice?: string;
-    gas?: number | string;
-    value?: number | string;
-    chainId?: number;
-    data?: string;
-    nonce?: number;
-    v?: string;
-    r?: string;
-    s?: string;
-}
+/** TODO - MOVE ALL BELOW TO WEB3-CORE ONCE FIXED CONFUSING WITH RETURN TYPES !!! */
```

Comment:
Should SignedTransaction and Sign not also be moved to the core module? or should the ```TODO``` comment be removed?

Reply:
not yet once it has been defined properly and is the expected outcome then we can move it. For now its only used in web3-eth-accounts so no point moving until the conversation we had on telegram is resolved. I added a note so that is fixed when that issue is sorted.